docs/src/pages/solid.mdx

```diff
@@ -115,10 +117,13 @@ The `@uploadthing/solid` package includes an "Uploader" component that you can s
 ```tsx copy
 // You need to import our styles for the button to look right. Best to import in the root /layout.tsx but this is fine
 import "@uploadthing/solid/styles.css";
+
 import { type VoidComponent } from "solid-js";
```

Comment:
Is there anything we can do to prefer 
```
import type { ... } from "some-package"
```
over
```
import { type ... } from "some-package"

Reply:
I don't know if there is a way to force you to change inline-type to separate ones, but with the current setup the auto fix is set to use separate-types.

Maybe the eslint-plugin-import has an option for this

Reply:
It does :) 
https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/consistent-type-specifier-style.md#prefer-top-level

Reply:
https://github.com/pingdotgg/uploadthing/assets/51714798/b48ef707-93f9-4de2-9e15-ebf82265f1eb

now only top-level are allowed